### PR TITLE
Gets rid of vali spear

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Gets rid of Vali spear and associated sprites.
## Why It's Good For The Game
Vali spear is a mistake, it allows a playstyle that should be limited entirely to 1 range to reach out to 2 range which allows vali to zone out xenos and hit things that it simply should not be able to hit, alongside making a play style defined by 'risk' be much much safer, even if you don't 2 hand it you can still use it 1 handed effectively.

This causes incredibly annoying gameplay for xenos because it means they *lose* out on what should be their niche which is melee.

## Changelog
:cl:
del: Vali spear has been removed.
/:cl:

### CODE REVIEW

**Comment by Barnet2** [comment]
image

**Comment by Lantekk** [comment]
Great PR

**Comment by novaepee** [comment]
I can't wait for a xeno with two tiles range.

**Comment by chizzy376** [comment]
Good riddance, 2.5 tiles of range is extremely annoying to play against.

**Comment by Strayed-Raven** [comment]
Now go improve melee code so you do not have a hard lock between inputs after you intentionally or unintentionally hit a xeno or anyone else with a melee weapon

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 1]
```diff
@@ -493,8 +491,6 @@
 			/obj/item/storage/holster/blade/machete/full = -1,
 			/obj/item/storage/holster/blade/machete/full_harvester = -1,
 			/obj/item/weapon/twohanded/spear/tactical = -1,
-			/obj/item/weapon/twohanded/spear/tactical/harvester = -1,
-			/obj/item/weapon/twohanded/glaive/harvester = -1,
```
Comment (Lumipharon) [inline code/game/objects/machinery/vending/marine_vending.dm:497]: why'd you get rid of both?

**Comment by Tommy-211** [comment]
Did you die

**Comment by Tommy-211** [comment]
This is just another xope merge, please grow up and maybe improve on yourself in the game to overcome this opponent of people with a 2 tile range

**Comment by Pariah919** [comment]
> die

I honestly don't think I've ever died to vali in general, so no?

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**